core/dbt/cli/flags.py

```diff
@@ -42,6 +43,7 @@
     "fail_fast",
     "indirect_selection",
     "store_failures",
+    "introspect",
```

Comment:
why we want to add introspect here?

Reply:
Do you mean removing it would cause things to fail? This doesn't feel right given that `introspect` is only a parameter for `compile` task command, and all parameter in this list is defiled both as `cli` click group level and a task command level.